README.md

```diff
@@ -1,34 +1,73 @@
-# bookay
 
-[![Build Status](https://github.com/jaynetics/bookay/workflows/tests/badge.svg)](https://github.com/jaynetics/bookay/actions)
+<br>
 
-bookay is a set of free, open-source tools for self-hosting your bookmarks.
+<div align = center>
 
-[
-  ![Screenshot thumbnail showing the webapp and browser plugin](
-    https://user-images.githubusercontent.com/10758879/113178657-1528d000-924f-11eb-9a02-2c0e4e504074.png
-  )
-](https://user-images.githubusercontent.com/10758879/113178042-77350580-924e-11eb-820f-298da2a2631d.png)
+[![Badge Build]][Actions]   [![Badge Heroku]][Heroku]
 
-## Why?
+<br>
 
-- a bookmarking service is useful if you use varying browsers or devices
-- there is currently no free service that supports nested folders
-- self-hosting your bookmarks offers the best privacy
+# Bookay
```

Comment:
please use a lowercase b - it is lowercase on purpose

Reply:
Sure, usually isn't so assumed it wasn't on purpose.